lambdas/nino-check/tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "noImplicitAny": true
   },
   "exclude": ["node_modules", "build", "coverage"],
-  "include": ["src", "tests"]
+  "include": ["src", "tests", "../common/src/database/retrieve-person-identity.ts", "../common/tests/database/retrieve-person-identity.test.ts"]
```

Comment:
Why do we need to do this?

Reply:
Oops, I thought I'd caught all of those - that was VSCode trying to be helpful when I was moving files around. Will remove 👍